sequence/model/modular.py

```diff
@@ -20,7 +20,7 @@ def __init__(
             self.vocabulary_size = custom_embeddings.shape[0]
             self.embedding_dim = custom_embeddings.shape[1]
             self.emb = nn.Embedding(*custom_embeddings.shape, _weight=custom_embeddings)  # type: ignore
-            self.emb.weight.requires_grad = False
+            self.emb.weight.requires_grad = True
```

Comment:
This line is redundant. The default is `True`. But do you want custom embeddings to be trainable?